documentation/service_admin/proc-deploying-address-space-controller.adoc

```diff
@@ -9,7 +9,7 @@ include::common/attributes.adoc[]
 
 The address space controller is responsible for creating the infrastructure used by address spaces.
 
-NOTE: To install {ProductName} on OpenShift, you must have `cluster-admin` access to set up the required roles for creating namespaces and managing resources in those namespaces; otherwise, you are restricted to a single address space. For more information about how to deploy without `cluster-admin` access, which restricts {ProductName} to a single address space, see xref:deploying-single-address-space-messaging[].
+NOTE: To install {ProductName} on OpenShift, you must have `cluster-admin` access to set up the required roles for creating namespaces and managing resources in those namespaces; otherwise, you are restricted to a single address space. For more information about how to deploy without `cluster-admin` access, which restricts {ProductName} to a single address space, see xref:deploying-single-address-space-{context}[].
```

Comment:
I think this was right before.  You want to put the link to the procedure in question

Reply:
The problem her is that `proc-deploying-address-space-controller` is part of `assembly-installing-manual-steps.adoc` which is part of both `assembly-installing-openshift` and `assembly-installing-kubernetes` which are 2 different contexts. This means that it needs to be linked to in 2 different contexts as well.

In the final doc, the kubernetes doc will end up linking to deploying-single-address-space-kube, and the openshift doc will end up linking to deploying-single-address-space-maas.

You can see the end result here: 

http://people.redhat.com/~ulilleen/maasdoc/master.html#deploying-address-space-controller-maas which links to http://people.redhat.com/~ulilleen/maasdoc/master.html#deploying-single-address-space-maas

http://people.redhat.com/~ulilleen/maasdoc/master.html#deploying-address-space-controller-kube which links to http://people.redhat.com/~ulilleen/maasdoc/master.html#deploying-single-address-space-kube

Am I using 'context' for the wrong thing here, and should instead create a different parameter?

On a separate note, I was wondering that if one procedure needs to be referenced in different contexts within the 'enmasse' documentation _and_ across products, the context needs to be 'appended', not overwritten. I.e. we should do `:context: {context}-maas` instead of `:context: maas` when assigning it.

Reply:
@ncbaratta Reading your other comment, I get the impression that as you say context should only be assigned in the master adoc, and not in the individual files. This means then that in order to have different contexts within the doc as we need for the installation docs, we should choose a different parameter.